prov/util/src/util_coll.c

```diff
@@ -1169,6 +1182,11 @@ int ofi_av_set(struct fid_av *av, struct fi_av_set_attr *attr,
 		av_set->fi_addr_count++;
 	}
 
+	ret = util_coll_mc_alloc(&av_set->coll_mc);
+	if (ret)
+		goto err2;
+	av_set->coll_mc->av_set = av_set;
+
 	av_set->av = util_av;
 	av_set->av_set_fid.ops = &util_av_set_ops;
```

Comment:
Here is where we would want to setup the av_set->av_set_fid.fid.ops, as mentioned above.